### PR TITLE
Ability to generate docs/pywbemcli/cmdshelp.rst with defined width

### PR DESCRIPTION
**Pull request by andy-maier**
See commit message.
To minimize changes, the width stays at 78 for now. It can be increased with a future PR.

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -153,7 +153,8 @@ Released: not yet
   (See issue #494)
 
 * Added support for adjusting the width of any help output to the terminal
-  width. (See issue #518)
+  width. The width can be set using the PYWBEMCLI_WIDTH env var.
```
Comment (KSchopmeyer) [inline docs/changes.rst]: Suggestion. TERMWIDITH rather than WIDITH

Comment (KSchopmeyer) [reply docs/changes.rst]: Misspelled widrh

Comment (andy-maier) [reply docs/changes.rst]: DONE - Changed PYWBEMCLI_WIDTH to PYWBEMCLI_TERMWIDTH.

**Comment by andy-maier** [comment]
Back to review needed.
I had to change some more things to make the tests succeed again.

**Comment by andy-maier** [comment]
Rebased the PR on PR #545 to get the dparse issue fixed.

**Pull request merged**